app/scripts/utils/interval-tree.js

```diff
@@ -113,13 +113,13 @@ function contains(point, node) {
    * @param {Number} point Point which should be checked.
    * @return {Boolean} True if point belongs to one of the intervals.
    */
-IntervalTree.prototype.contains = function (point) {
+IntervalTree.prototype.contains = function _contains(point) {
```

Comment:
Same as the question above?

Reply:
Going to leave this as is... but push back if you think we're mistaken!